### ISSUE TITLE
ROI does not ignore on-drop callback if in an invalid state

### ISSUE THREAD

**Issue by papr** [pupil-labs/pupil#1825]
`on_click` traceback
```
Traceback (most recent call last):
  File "/Users/papr/work/pupil/pupil_src/launchables/eye.py", line 633, in eye
    if plugin.on_click(pos, button, action):
  File "/Users/papr/work/pupil/pupil_src/shared_modules/roi.py", line 287, in on_click
    clicked_handle = self.get_handle_at(pos)
  File "/Users/papr/work/pupil/pupil_src/shared_modules/roi.py", line 250, in get_handle_at
    for handle in self._all_points.keys():
AttributeError: 'Roi' object has no attribute '_all_points'
```

`Roi._all_points` is set in `Roi.reset_points()` but only if `not Roi.model.is_invalid()`. If the restored session settings are invalid - which should not happen in the first place - clicking on the eye window will cause a crash (see above).

Note: This happend while there was no active headset connected.

**Comment by papr** [comment]
Understood. I agree with this solution.

**Comment by papr** [comment]
Fixed by #1827
